components/BasePagination.vue

```
@@ -0,0 +1,57 @@
+  goToNext,
+} = defineProps<{
+  currentPage: number
+  totalPages: number
+  hasPrevious: boolean
+  hasNext: boolean
+  goToPrevious: () => void
+  goToNext: () => void
+}>()
+
+const appConfig = useAppConfig()
+const { showPageNumbers } = appConfig.site.pagination
```

ESLint

[error] 'showPageNumbers' is assigned a value but never used.

(no-unused-vars)